templates/mon_mysql.sql.j2

```diff
@@ -6,7 +6,7 @@ SET foreign_key_checks = 0;
 CREATE TABLE `alarm` (
   `id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL,
   `alarm_definition_id` varchar(36) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
-  `state` enum('UNDETERMINED','OK','ALARM') COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
I haven't researched the issue at all and don't know what is best. @roland-hochmuth do you want to take a moment to look at this and give your thoughts?